R/DataSetList.R

```diff
@@ -396,48 +396,51 @@ arrange.DataSetList <- function(dsl, ...) {
 }
 
 #' @rdname get_ERT
-#' @param algorithm Which algorithms in the DataSetList to consider.
+#' @param algorithm DEPRECATED, will be removed in next release. Which algorithms in the DataSetList to consider. 
 #' @export
 #'
 get_ERT.DataSetList <-
-  function(ds, ftarget, algorithm = 'all', ...) {
+  function(ds, ftarget, budget = NULL, algorithm = 'all', ...) {
```

Comment:
I prefer to have the left and right values of an assignment on the same line. But, are you aware of any coding style for R? (probably there is no standard, I guess)

Reply:
Yes, I agree that this should be on the same line (don't remember why it was on separate lines for these functions)